resources/css/app.css

```diff
@@ -17,15 +17,3 @@ pre.shiki {
 pre.shiki code {
     @apply max-w-[200px] block;
 }
-
-.markdown-text {
```

Comment:
I didn't like this here anyways 🤣

Reply:
Me neither, but I just found out about it in the Tailwind documentation! 😂